Gemfile

```diff
@@ -65,6 +65,8 @@ gem 'formvalidation-rails'
 
 gem 'data-confirm-modal', github: 'ifad/data-confirm-modal'
 
+gem 'jquery-turbolinks'
```

Comment:
Cool - I'm glad to see you're giving Turbolinks a try. I'm working on an app and using Turbolinks throughout as a bit of an experiment ( http://nonstop.qa ). So far, I really like the results, except for the progress bar. I want to create my own that's less dominant in the layout.